lib/config/presets/bitbucket-server/index.ts

```diff
@@ -17,11 +17,16 @@ const http = new BitbucketServerHttp();
 export async function fetchJSONFile(
   repo: string,
   fileName: string,
-  endpoint: string
+  endpoint: string,
+  branchOrTag?: string
 ): Promise<Preset> {
   const [projectKey, repositorySlug] = repo.split('/');
   setBaseUrl(endpoint);
-  const url = `rest/api/1.0/projects/${projectKey}/repos/${repositorySlug}/browse/${fileName}?limit=20000`;
+  let url = `rest/api/1.0/projects/${projectKey}/repos/${repositorySlug}/browse/${fileName}?limit=20000`;
+  if (branchOrTag) {
+    url += '&at=' + encodeURIComponent(`refs/heads/${branchOrTag}`);
```

Comment:
does it work with git tags too?

Reply:
@viceice I tried to fix this so it should work with branch names and tags as well.